File: SwimmingTurtle/src/CaveEscape1.pas
program GameMain;
uses SwinGame, sgTypes, sgTimers, sgSprites, sysUtils;

procedure Main();
begin
  OpenGraphicsWindow('Cave Escape', 432, 768);
	OpenAudio();
  LoadResources();

  repeat // The game loop...
    ProcessEvents();

    ClearScreen(ColorWhite);
    RefreshScreen()

  until WindowCloseRequested();
end;

begin
  Main();
end.

File: CaveEscapeC/src/cave_escape_1.cpp
#include <stdio.h>
#include "SwinGame.h"

int main()
{
    open_graphics_window("Cave Escape", 432, 768);

    do // The game loop...
    {
      process_events();

      clear_screen(COLOR_WHITE);
      refresh_screen();

    } while(!window_close_requested());

    return 0;
}
